src/Bedrock.Framework.Experimental/Protocols/Http1RequestMessageReader.cs

```diff
@@ -42,8 +42,16 @@ public bool TryParseMessage(in ReadOnlySequence<byte> input, ref SequencePositio
                         return false;
                     }
 
+                    //First char must be / for relative URI
+                    if (path[0] == (byte)'/')
```

Comment:
The `byte` casting can be cached.

Reply:
I know this is old.  But @adamradocz , won't the compiler just have this as a literal?  No need to cache something that isn't an instance, right?

Reply:
Also, @nickbreese, couldn't a relative URIL start with a .?  So, like "./Blah"?  Or, it could even be "images/image1.jpg"?   If any of these are true, this will not work.

Reply:
@smeddows, yes, you're right.